searchcore/src/tests/proton/documentdb/documentbucketmover/documentbucketmover_test.cpp

```diff
@@ -1219,6 +1223,25 @@ TEST_F("explicitly active not ready bucket can be moved to ready even if node is
     EXPECT_EQUAL(f._notReady.bucket(3), f.bucketsModified()[0]);
 }
 
+TEST_F("require that notifyCreateBucket works", ControllerFixture)
```

Comment:
Consider changing test name to what is expected when it is called, as specified in the commit message ("... causes bucket to be reconsidered by job")